src/aws/eventBridgeBase.ts

```diff
@@ -27,18 +28,18 @@ export class EventBridgeBase {
     try {
       const output: PutEventsCommandOutput = await this.client.send(command);
       if (output.FailedEntryCount) {
+        serverLogger.error(failedEventError);
```

Comment:
definitely not a blocker, but what do you think about adding more structure to these calls as a general practice? something like `serverLogger.error('failed putting events to event bridge', { failedEventError, originalError: error});`

Reply:
I agree, could we follow up with that type of stuff?

Reply:
yeah, 100%. no reason to block this PR on that.